microsoft-edge/devtools-guide-chromium.md

```diff
@@ -12,104 +12,140 @@ keywords: microsoft edge, web development, f12 tools, devtools
 
 # Microsoft Edge (Chromium) Developer Tools  
 
-In the next version of Microsoft Edge, we will be adopting the Chromium open source project to create better web compatibility and less fragmentation of different underlying web platforms.  This change should make it easier for you to build and test your web sites in Microsoft Edge and ensure they will still work as expected even if your users are browsing in a different Chromium-based browser, like Google Chrome, Vivaldi, Opera, or Brave.  
+Microsoft Edge has adopted the Chromium open source project to create better web compatibility and less fragmentation of different underlying web platforms.  This change should make it easier for you to build and test your websites in Microsoft Edge and ensure that each works as expected even while your users are browsing in a different Chromium-based browser \(such as Google Chrome, Vivaldi, Opera, or Brave\).  
 
-As the web has grown in usage across an ever-widening array of device types, the complexity and overhead involved in testing web sites has exploded. Since web developers— particularly those at small companies— need to test so many different systems, it's nearly impossible to ensure that interesting sites will work well across all device types and all browsers. With the next version of Microsoft Edge, we hope to simplify this matrix for you by aligning the Microsoft Edge web platform with other Chromium-based browsers and providing a best-in-class developer tooling experience, both inside the browser and with the other developer tools you use every day, like Visual Studio Code.
+As the web has grown in usage across an ever-widening array of device types, the complexity and overhead involved in testing websites has exploded. Since web developers \(particularly those at small companies\) must test against so many different systems, you may find it nearly impossible to ensure that sites work well on all device types and all browsers.  Now that Microsoft Edge is based on Chromium, The Microsoft Edge team has simplified the matrix by aligning the Microsoft Edge web platform with other Chromium-based browsers and provided a best-in-class developer tooling experience, both inside the browser and with the other developer tools you use every day \(such as Visual Studio Code\).  
```

Comment:
```suggestion
As the web has grown in usage across an ever-widening array of device types, the complexity and overhead involved in testing websites has exploded. Since web developers \(particularly those at small companies\) must test against so many different systems, you may find it nearly impossible to ensure that sites work well on all device types and all browsers.  Now that Microsoft Edge is based on Chromium, the Microsoft Edge team has simplified the matrix by aligning the Microsoft Edge web platform with other Chromium-based browsers and provided a best-in-class developer tooling experience, both inside the browser and with the other developer tools you use every day \(such as Visual Studio Code\).  
```